docker-compose/docker-whisk-controller.env

```diff
@@ -36,7 +36,7 @@ LIMITS_ACTIONS_SEQUENCE_MAXLENGTH=20
 
 CONFIG_whisk_loadbalancer_blackboxFraction=0.1
 CONFIG_kamon_statsd_port=8125
-CONFIG_whisk_loadbalancer_invokerBusyThreshold=16
+CONFIG_whisk_loadbalancer_invokerUserMemory=1024 m
```

Comment:
Also add `CONFIG_whisk_containerPool_userMemory="1024 m"`

Reply:
Thanks Carlos, I was trying to find out what the other one was